extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -38,10 +38,14 @@ function __construct( $key, $archive = 0 ) {
 				$this->mEventType = self::UNDELETE_CATEGORY_INT;
 				break;
 		}
-
+		$geo = json_decode($this->app->wg->Request->getCookie( "Geo", "" ));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)